edit-post/store/selectors.js

```diff
@@ -108,6 +108,20 @@ export function isFeatureActive( state, feature ) {
 	return !! state.preferences.features[ feature ];
 }
 
+/**
+ * Returns true if the the plugin item is pinned to the header.
+ * When the value is not set it defaults to true.
+ *
+ * @param  {Object}  state      Global application state.
+ * @param  {string}  pluginName Plugin item name.
```

Comment:
Similar nit: Documentation standards recommend `@param` and `@return` to be separate groups, no need to align between the two.

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#functions